src/high-level/include/iDynTree/KinDynComputations.h

```diff
@@ -396,6 +532,17 @@ class KinDynComputations {
     iDynTree::Transform getRelativeTransform(const iDynTree::FrameIndex refFrameIndex,
                                              const iDynTree::FrameIndex frameIndex);
 
+    /**
+     * Return the transform where the frame is the frame
+     * specified by frameIndex, and the reference frame is the one specified
+     * by refFrameIndex (refFrame_H_frame).
+     * @warning the Span and the MatrixView objects should point an already existing memory. Memory allocation and resizing cannot be achieved with this kind of objects.
```

Comment:
See the comment in the `getWorldTransform` method.
Also, be consistent with the documentation. 
I _think_ it is known that a span cannot be resized, but if you think the user should know this, add it as class comment as you have lots of methods taking Spans/MatrixViews